docs/09_tutorials/10_nested_containers/index.md

```diff
@@ -0,0 +1,206 @@
+| map | [supported](https://github.com/EOSIO/eosio.cdt/blob/46cfbd2c49a98a425fb0cc39243c3813c62ebef3/tests/toolchain/build-pass/nestcontn2kv.cpp#L859) | [supported](https://github.com/EOSIO/eosio.cdt/blob/46cfbd2c49a98a425fb0cc39243c3813c62ebef3/tests/toolchain/build-pass/nestcontn2kv.cpp#L893) | [supported](https://github.com/EOSIO/eosio.cdt/blob/46cfbd2c49a98a425fb0cc39243c3813c62ebef3/tests/toolchain/build-pass/nestcontn2kv.cpp#L926) | [supported](https://github.com/EOSIO/eosio.cdt/blob/46cfbd2c49a98a425fb0cc39243c3813c62ebef3/tests/toolchain/build-pass/nestcontn2kv.cpp#L958) | [supported](https://github.com/EOSIO/eosio.cdt/blob/46cfbd2c49a98a425fb0cc39243c3813c62ebef3/tests/toolchain/build-pass/nestcontn2kv.cpp#L993) | [supported](https://github.com/EOSIO/eosio.cdt/blob/46cfbd2c49a98a425fb0cc39243c3813c62ebef3/tests/toolchain/build-pass/tupletestkv.cpp#L211)  |
+| pair | [supported](https://github.com/EOSIO/eosio.cdt/blob/46cfbd2c49a98a425fb0cc39243c3813c62ebef3/tests/toolchain/build-pass/nestcontn2a.cpp#L998) | [supported](https://github.com/EOSIO/eosio.cdt/blob/46cfbd2c49a98a425fb0cc39243c3813c62ebef3/tests/toolchain/build-pass/nestcontn2a.cpp#L1022) | [supported](https://github.com/EOSIO/eosio.cdt/blob/46cfbd2c49a98a425fb0cc39243c3813c62ebef3/tests/toolchain/build-pass/nestcontn2a.cpp#L1048) | [supported](https://github.com/EOSIO/eosio.cdt/blob/46cfbd2c49a98a425fb0cc39243c3813c62ebef3/tests/toolchain/build-pass/nestcontn2a.cpp#L1079) | [supported](https://github.com/EOSIO/eosio.cdt/blob/46cfbd2c49a98a425fb0cc39243c3813c62ebef3/tests/toolchain/build-pass/nestcontn2a.cpp#L1105)  | [supported](https://github.com/EOSIO/eosio.cdt/blob/46cfbd2c49a98a425fb0cc39243c3813c62ebef3/tests/toolchain/build-pass/tupletestkv.cpp#L242) |
+| tuple |  [supported](https://github.com/EOSIO/eosio.cdt/blob/46cfbd2c49a98a425fb0cc39243c3813c62ebef3/tests/toolchain/build-pass/tupletestkv.cpp#L327) |  [supported](https://github.com/EOSIO/eosio.cdt/blob/46cfbd2c49a98a425fb0cc39243c3813c62ebef3/tests/toolchain/build-pass/tupletestkv.cpp#L296) |  [supported](https://github.com/EOSIO/eosio.cdt/blob/46cfbd2c49a98a425fb0cc39243c3813c62ebef3/tests/toolchain/build-pass/tupletestkv.cpp#L363) |  [supported](https://github.com/EOSIO/eosio.cdt/blob/46cfbd2c49a98a425fb0cc39243c3813c62ebef3/tests/toolchain/build-pass/tupletestkv.cpp#L402) |  [supported](https://github.com/EOSIO/eosio.cdt/blob/46cfbd2c49a98a425fb0cc39243c3813c62ebef3/tests/toolchain/build-pass/tupletestkv.cpp#L434)  | [supported](https://github.com/EOSIO/eosio.cdt/blob/46cfbd2c49a98a425fb0cc39243c3813c62ebef3/tests/toolchain/build-pass/tupletestkv.cpp#L264) |
+
```

Comment:
In above table, each cell (say vector of optional) actually has TWO links: one points to how to use it in a multi-index table, the 2nd link points to how to use it in kv::table, if you want to show the links in a multi-index table, then the first row (and all other rows) should use links in nestcontn2a.cpp and tupletest.cpp (NOT tupletestkv.cpp).

Also I have removed some unsuitable comments in tupletest.cpp and tupletestkv.cpp in https://github.com/EOSIO/eosio.cdt/pull/1229, so line numbers of those two files should be changed.

I suggest using TWO tables, with table 1 using lines/links in nestcontn2a.cpp and tupletest.cpp  for multi-index table, and table 2 using lines/links in nestcontn2kv.cpp and tupletestkv.cpp for the value part of kv::map. And then change the description here like,  matrix 1  is applicable to **eosio::multi_index**, matrix 2  is applicable to the **value part of eosio::kv::map**.